### PR TITLE
Temporary module to remove payment dupes

### PR DESCRIPTION
**Pull request by clarissalimab**
Part of https://github.com/CommitChange/tix/issues/3553

### CODE REVIEW

File: lib/payment_dupes.rb [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+                                    reasons.join(', ')
+                                ]
+                            end
+                        end
+                    end
+                end
+            end
+            File.write('payment_dupes.csv', Format::Csv.from_array(duplicate_payments))
+            File.write('couldnt_delete.csv', Format::Csv.from_array(couldnt_delete))
+            raise 'done'
```
Comment (clarissalimab) [inline lib/payment_dupes.rb]: Once it gets approved I'm going to remove this line.

File: db/migrate/20220115171649_add_matched_to_payments.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddMatchedToPayments < ActiveRecord::Migration
+  def change
+    add_column :payments, :matched, :boolean, default: false
```
Comment (wwahammy) [inline db/migrate/20220115171649_add_matched_to_payments.rb]: Please put the matched field on another model and use a has_one reference from Payment to the table and a belongs_to from the table to Payment. Adding fields to a model is not good practice and it breaks all of the tests

**Review by wwahammy — CHANGES_REQUESTED** [review]
We can't merge as is but I guess this is fine for just running the tests required of the CommitChange/tix#3553

**Review by wwahammy — APPROVED** [review]
Seems good. Again, it's hard to tell. Nothing jumps out at me as wrong.

**Pull request merged**